requirements.txt

```diff
@@ -34,7 +34,7 @@ cssselect==0.9.1
 splinter==0.7.5
 tblib==1.3.0
 selenium==2.53.6
-django-filer==1.2.5.post1
```

Comment:
Cosa succede alla vostra versione modificata di django-filer in caso di aggiornamenti di sicurezza al modulo madre? Ad esempio, in caso di vulnerabilita', riuscite a garantire il supporto? Se la risposta e' no, e non possiamo mai piu' aggiornare il modulo, chiaramente dobbiamo trovare una soluzione migliore.

Vedo che ad oggi la vostra versione modificata di django-filer e' 1200 commit, circa 2 anni, indietro rispetto alla upstream. Immagino che la maggior parte siano modifiche che non ci importano, ma siamo sicuri che non vi sia alcun fix di sicurezza?